solr/core/src/java/org/apache/solr/core/SolrXmlConfig.java

```diff
@@ -548,6 +548,10 @@ private static MetricsConfig getMetricsConfig(XmlConfigFile config) {
       log.info("Metrics collection is disabled.");
       return builder.build();
     }
+    node = config.getNode("solr/metrics/coreLevelMetricsEnabled", false);
```

Comment:
That argument is actually if it should error if missing from solr.xml, which we want to be false. That configuration in solr.xml will generally be absent by default, in which case coreLevelMetricsEnabled will default to true because that is the default here: https://github.com/fullstorydev/lucene-solr/blob/7b0eae09789d54a66d9b8efb884ff34150ef1d5c/solr/core/src/java/org/apache/solr/core/MetricsConfig.java#L168.